presto-product-tests/src/main/java/com/facebook/presto/tests/TestGroups.java

```diff
@@ -52,6 +52,7 @@
     public static final String AUTHORIZATION = "authorization";
     public static final String POST_HIVE_1_0_1 = "post_hive_1_0_1";
     public static final String PREPARED_STATEMENTS = "prepared_statements";
+    public static final String BIG_QUERY = "big_query";
```

Comment:
I think we have had such a group already.
Even presto-product-tests readme refer to it: https://github.com/prestodb/presto/tree/master/presto-product-tests